src/h5cli/explorer.py

```diff
@@ -12,6 +12,7 @@ def __init__(self, h5_obj):
 
     @staticmethod
     def from_file(filename, mode="r"):
+        filename = os.path.expanduser(filename)
```

Comment:
Why not just go with a full os.path.abspath() ?

Reply:
abspath doesnt resolve the tilde, it just tacks the current directory in front then calls normpath, which removes any '..' or '//' and the like
absolute paths are not needed here,relative is fine,but want to expand the tilde

Reply:
huh, didn't realize that abspath wasn't smart enough to expand the tilde as well. Good point though.